### ISSUE TITLE
Invalid error position for redeclared symbol

### ISSUE THREAD

**Issue by rdulmina** [ballerina-platform/ballerina-lang#26600]
**Description:**
For the following code 
```
[int ,float, string] [a, b, c] = [1, 2.5, "Dulmina"];
float b = 6;
```
The compiler incorrectly identifies `b` in the tuple variable as the redeclared one

`error: .::module_tuple_var_decl_negetive.bal:1:26: redeclared symbol 'b'`

Where the actual redeclared one is in the second line

**Steps to reproduce:**
Module tuple binding pattern support added by #26599

**Suggested Labels:**

- Team/Compiler FE
- Type/Bug

**Comment by rdulmina** [comment]
Fixed in #26599 itself.

**Comment by mohanvive** [comment]
Closing the issue as per the above comment.
